File: Container_Loop/4.py
input_id = input("아이디를 입력해주세요: ")

# real_egoing  = "egoing"
# real_k8805 = "k8805"
members = ['egoing', 'k8805']
members.append('leezche')

# if real_egoing == input_id:
#     print("Hello, egoing!")
# elif real_k8805 == input_id:
#     print("Hello, k8805!")
# else:
#     print("Who are you?")
for member in members:
    if member == input_id:
        print("Hello, " + member + "!")
        import sys
        sys.exit()  #프로그램 종료
print("Who are you?")

File: Container_Loop/4.rb
print("입력해주세요: ")
input_id = gets.chomp()
# real_egoing  = "egoing"
# real_k8805 = "k8805"
members = ['egoing', 'k8805']
members << 'leezche'

# if real_egoing == input_id
#   puts("Hello, egoing!")
# elsif real_k8805 == input_id
#   puts("Hello, real_k8805!")
# else
#   puts("Who are you?")
# end
for member in members do
  if member == input_id
    puts("Hello, "+member+"!")
    exit
  end
end
puts("Who are you?")
